dts/arm/nxp/nxp_rt.dtsi

```diff
@@ -231,9 +231,12 @@
 			status = "disabled";
 		};
 
-		iomuxc: iomuxc@401f8000 {
+		pinctrl: pinctrl@401f8000 {
+			compatible = "nxp,mcux-rt-pinctrl";
 			reg = <0x401f8000 0x4000>;
-			label = "PINMUX_0";
+			#address-cells = <1>;
```

Comment:
Do we need `address-cells` and `size-cells`? The dts parser was giving me a warning that they were unused

Reply:
yes, they are not used for now, I just keep it here and maybe we need use this.